src/Data/TypeRepMap/Internal.hs

```diff
@@ -170,7 +169,12 @@ prop> member @a (one (x :: f a)) == True
 
 -}
 one :: forall a f . Typeable a => f a -> TypeRepMap f
-one x = insert x empty
+one x = TypeRepMap (primArrayFromListN 1 [fa])
+                   (primArrayFromListN 1 [fb])
+                   (pure v)
+                   (pure k)
```

Comment:
I think, using type applications to create arrays with `pure` will help here for readability in the future
```suggestion
                   (pure @Array v)
                   (pure @Array k)
```